src/android/HttpRequests.java

```diff
@@ -0,0 +1,50 @@
+package com.cowbell.cordova.geofence;
+
+import android.util.Log;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class HttpRequests {
```

Comment:
Great idea to introduce something like that, but should be in separate PR, and it should be more configurable from cordova application.